### DISCUSSION TITLE
Randomize users in People widget on organization page

### DISCUSSION THREAD

**Discussion by wrooot** [community/community#22265] [New to GitHub]
<p>There is no simple feedback form anywhere on GitHub and i’m not sure if this forum is right place for a trivial suggestion. But i have noticed that People widget on organization page is always listing same users in alphabetical order. It should be more fair and fun to see randomized picks in this place, so users with usernames starting with w (like myself) would get some spotlight too :smiley:</p>

**Comment by lee-dohm** [comment]
<p>I’m not sure I’m understanding completely what you’re suggesting. As an example, I’m looking at the <a href="https://github.com/orgs/rails/people" rel="nofollow noopener">People widget for the Rails organization</a>. I see in that paginated list all members of the Rails org that have elected to list themselves publicly. Yes, it is alphabetical, but it doesn’t exclude anyone (that haven’t elected to be private) because you can always <a href="https://github.com/orgs/rails/people?page=2" rel="nofollow noopener">navigate to successive pages.</a></p>
<p>Am I understanding which page you’re talking about correctly?</p>

**Comment by wrooot** [comment]
<p>I was talking about a widget on this page <a href="https://github.com/rails" rel="nofollow noopener">https://github.com/rails</a></p>
<p>It always shows first 16 members of organization.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Ah ok, thanks for clarifying :+1:</p>
<p>You can currently submit any feature requests via our contact form: <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a>. Longer-term we want to take feature requests such as these directly through the Forum but until that’s in place we’re going to keep on with our current system.</p>
<p>Thanks again for the feedback and ideas!</p>

**Comment by wrooot** [comment]
<p>Thanks, will submit it via that form.</p>

**Comment by wrooot** [comment]
<p>Still same alphabetical order…</p>
